travis-builds/build_imgs.sh

```diff
@@ -34,23 +34,23 @@ function copy_dirs {
 
 function build_base_img {
   pushd base
-  docker build -t base .
+  docker build -t ceph/base .
```

Comment:
This obviously breaks Travis but I guess we don't care as this should replace Travis in the long term.